packages/experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -242,9 +239,13 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
                   onClick={_onInputClick}
                   onInputValueChange={_onInputChange}
                   /* eslint-enable react/jsx-no-bind */
-                  aria-activedescendant={activeDescendant}
-                  aria-owns={isExpanded ? 'suggestion-list' : undefined}
-                  aria-expanded={isExpanded}
+                  aria-activedescendant={
+                    isSuggestionsShown && focusItemIndex >= 0
+                      ? 'FloatingSuggestionsItemId-' + focusItemIndex
+                      : undefined
+                  }
+                  aria-owns={isSuggestionsShown ? 'suggestion-list' : undefined}
+                  aria-expanded={isSuggestionsShown}
                   aria-haspopup="true"
                   role="combobox"
```

Comment:
Aria 1.1 guidance is to have `role='combobox'` on a container and then have the typable `<input>` retain its textbox role

e.g., https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

Reply:
Fixed in the new iteration